src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -1385,7 +1385,7 @@ void
 OverlayImpl::unsquelch(PublicKey const& validator, Peer::id_t id) const
 {
     if (auto peer = findPeerByShortID(id);
-        peer && app_.config().REDUCE_RELAY_SQUELCH)
+        peer && app_.config().VP_REDUCE_RELAY_SQUELCH)
```

Comment:
This is useful for debugging really. Neither `squelch()` or `unsquelch()` will be ever called if the feature is disabled at the protocol level.  If the feature is enabled and `VP_REDUCE_RELAY_SQUELCH` is disabled then squelch/unsquelch are never sent to peers but the logic otherwise works. For instance can connect to the mainnet with the feature enabled but squelch disabled and see all the interactions but without affecting any connected peers.

Reply:
So `VP_REDUCE_RELAY_SQUELCH` is only used for testing, right? 
Do we plan to remove it after testing? Either way, we probably should document it more clearly in config.h.

Reply:
It is useful for testing and I'd like to keep since it provides an ability to test the feature's logic without affecting the relay. Will update the comments.